tools/runtime-codegen/Cargo.toml

```diff
@@ -16,7 +16,9 @@ codec = { package = "parity-scale-codec", version = "3.1.5", default-features =
 color-eyre = "0.6.1"
 proc-macro2 = "1.0.56"
 quote = "1.0.28"
-syn = "1.0"
-subxt-codegen = { git = "https://github.com/paritytech/subxt", branch = "master", default-features = false, features = [] }
+subxt-codegen = { git = "https://github.com/paritytech/subxt", branch = "master", default-features = false, features = ["fetch-metadata"] }
 wasm-loader = { git = "https://github.com/chevdor/subwasm", branch = "master" }
 wasm-testbed = { git = "https://github.com/chevdor/subwasm", branch = "master" }
+
+[patch."https://github.com/paritytech/subxt"]
+subxt-codegen = { path = "../../../subxt/codegen" }
```

Comment:
Is that intended? I mean - I have no local copy :)

Reply:
Ups, this was for local debugging. Sorry. Removed.